pallets/gear/src/weights.rs

```diff
@@ -2301,10 +2321,18 @@ impl WeightInfo for () {
         // Proof Size summary in bytes:
         //  Measured:  `0`
         //  Estimated: `0`
-        // Minimum execution time: 83_351_000 picoseconds.
-        Weight::from_parts(111_337_720, 0)
-            // Standard Error: 455_985
-            .saturating_add(Weight::from_parts(163_069_244, 0).saturating_mul(r.into()))
+        // Minimum execution time: 91_287_000 picoseconds.
+        Weight::from_parts(93_643_000, 0)
+            // Standard Error: 401_184
+            .saturating_add(Weight::from_parts(462_939_312, 0).saturating_mul(r.into()))
+    }
+    /// The range of component `p` is `[1, 512]`.
+    fn alloc_per_page(_p: u32, ) -> Weight {
```

Comment:
Also very strange that alloc per page is bigger than alloc, because
``` 
alloc_bench = 80 * (const_alloc_cost + alloc_per_page_cost + grow_cost)
alloc_per_page_bench = 80 * alloc_per_page_cost
```
So you can see it must be less than.

Reply:
I have on my MacOs all weights as I expected. Where const_alloc_cost is much bigger than alloc_per_page_cost